cloud/src/rate-limiter/rate_limiter.h

```diff
@@ -35,12 +38,32 @@ class RateLimiter {
 public:
     RateLimiter() = default;
     ~RateLimiter() = default;
+
     void init(google::protobuf::Service* service);
+
     std::shared_ptr<RpcRateLimiter> get_rpc_rate_limiter(const std::string& rpc_name);
 
+    void for_each_rpc_limiter(
+            std::function<void(std::string_view, std::shared_ptr<RpcRateLimiter>)> cb);
+
+    // set global default rate limit, will not infulence rpc and instance specific qps limit setting
+    bool set_rate_limit(int64_t qps_limit);
+
+    // set rpc level rate limit, will not infulence instance specific qps limit setting
+    bool set_rate_limit(int64_t qps_limit, const std::string& rpc_name);
+
+    // set instance level rate limit for specific rpc
+    bool set_rate_limit(int64_t qps_limit, const std::string& rpc_name,
+                        const std::string& instance_id);
+
+    // set instance level rate limit globally, will influence settings for the same instance of specific rpc
+    bool set_instance_rate_limit(int64_t qps_limit, const std::string& instance_id);
+
 private:
     // rpc_name -> RpcRateLimiter
     std::unordered_map<std::string, std::shared_ptr<RpcRateLimiter>> limiters_;
+    std::unordered_set<std::string> rpc_with_specific_limit_;
```

Comment:
add comment, it seems to be "This collection contains the RPCs that have had a limit set by the specified name"